### DISCUSSION TITLE
revealjs and observable table input: how to stretch

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#830] [Q&A] [answered]
Hi,
I'm using the code below to render a CSV using an [observable Table Input](https://observablehq.com/@observablehq/input-table).

The rendered table is not stretched to fill the free entire height. Is there a way to stretch it?

Thank you 

![image](https://user-images.githubusercontent.com/30607/167308860-b3c6e0d6-ffae-4ea7-9389-2c362a4d7e1b.png)

~~~
---
title: "OJS"
format:
  revealjs:
    slide-number: true
    footer: <https://quarto.org>
    navigation-mode: vertical
---

## Table

```{ojs}
input = FileAttachment("https://raw.githubusercontent.com/mwaskom/seaborn-data/master/penguins.csv").csv({typed: true})
Inputs.table(input)
```
~~~

**Comment by aborruso** [comment, marked as answer]
Hi,
I'm adding a useful note: I can use the ojs table input options, and one of it is height.

```
Inputs.table(input, {height: 500})
```

And in this way I have something similar the result I want.

**Reply by cscheid** [reply]
Thank you for figuring this out!

In general, you should expect OJS stdlib calls to require some manual tweaking, since the CSS they emit usually won't match ours.
